x-pack/plugins/security_solution/public/common/components/event_details/event_details.tsx

```diff
@@ -288,7 +288,7 @@ const EventDetailsComponent: React.FC<Props> = ({
                     isReadOnly,
                   }}
                   goToTable={goToTableTab}
-                  investigationFields={maybeRule?.investigation_fields ?? []}
+                  investigationFields={maybeRule?.investigation_fields?.fields ?? []}
```

Comment:
nit: `investigation_fields?.fields` - _fields.fields feels a bit repetitive. Dp not have a strong opinion on that, but maybe `investigation_fields.keys` or `investigation_options.fields` would be more suitable